apps/server/src/apps/server.app.ts

```diff
@@ -12,14 +12,14 @@ import { MikroORM } from '@mikro-orm/core';
 import { Logger } from '@nestjs/common';
 import { enableOpenApiDocs } from '@shared/controller/swagger';
 import { Mail, MailService } from '@shared/infra/mail';
+import { AccountService } from '@src/modules/account/services/account.service';
```

Comment:
Not part of this PR, but it looks wrong that the service is needed on this place instead of the modul.